Source/JavaScriptCore/jit/JITPlan.cpp

```diff
@@ -180,15 +180,15 @@ void JITPlan::compileInThread(JITWorklistThread* thread)
         StringPrintStream stream;
         stream.print(m_mode, " ", *m_codeBlock, " instructions size = ", m_codeBlock->instructionsSize());
         signpostMessage = stream.toCString();
-        WTFBeginSignpost(this, JSCJITCompiler, "%" PUBLIC_LOG_STRING, signpostMessage.data());
+        WTFBeginSignpost(this, JSCJITCompiler, "%" PUBLIC_LOG_STRING, signpostMessage.data() ?: "(nullptr)");
```

Comment:
Haven't seen that shorthand for the ternary operator in WebKit so far, doesn't trigger a style violation, but still slightly awkward to my untrained eye :-)

Reply:
I think this is GCC extension, and we should avoid it.